dspace/dspace.dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM janx/ubuntu-dev
+RUN sudo apt-get update -q \
+  && sudo apt-get upgrade -qy \
+  && sudo apt-get install -qy \
+     ant \
+     maven \
+     postgresql \
+     tomcat8
+
+RUN git clone https://github.com/dspace/dspace /home/user/dspace
+WORKDIR /home/user/dspace
+
+COPY local.cfg /home/user/dspace/dspace/config
```

Comment:
Nit: Please add a trailing `/` after `config/` to indicate that it should be a folder (and in case it doesn't exist, that `local.cfg` should never be renamed `config` accidentally).